source/fundamentals/transactions.txt

```diff
@@ -273,28 +359,34 @@ you want to run on that session.
 
 The code and comments in the ``catch`` block demonstrate how you can identify
 the server transaction errors and where you can place your logic to handle
-them. Make sure to include the ``MongoError`` type from the driver in your
-code as shown in the following sample import statement:
+them. Make sure to import the ``MongoError`` type from the driver
+package as shown in the following sample import statement:
 
 .. code-block:: javascript
 
    const { MongoError, MongoClient } = require('mongodb');
 
-See the :ref:`Payment Transaction Result <nodejs-transaction-example-payment-result>`
-section to see what your collections should contain after you run the
-transaction.
+The :ref:`Transaction Result <nodejs-transaction-example-payment-result>`
+section describes what data your collections should contain after you
+successfully perform the transaction.
+
+.. tip::
+   
+   To view an example that uses the Convenient Transaction API, see the
+   :ref:`node-usage-convenient-txn` usage example.
 
 .. _nodejs-transaction-example-payment-result:
 
-Payment Transaction Result
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+Transaction Result
```

Comment:
Suggestion:

I think "Transaction" should be plural since the section describes two results -- one in which it was successful and another in which an error interrupts the transaction.

```suggestion
Transaction Results
```